app/javascript/mastodon/components/display_name.js

```diff
@@ -3,6 +3,7 @@ import ImmutablePropTypes from 'react-immutable-proptypes';
 import PropTypes from 'prop-types';
 import { autoPlayGif } from 'mastodon/initial_state';
 import Skeleton from 'mastodon/components/skeleton';
+import punycode from 'punycode';
```

Comment:
The Node.js built-in module `punycode` is deprecated.

see https://nodejs.org/dist/latest-v14.x/docs/api/punycode.html

Reply:
This is the one from npm though, as specified in the package.json.

Reply:
Ah, I get what you mean. Fixed it so it explicitly references the userland implementation, and fixed an instance where previously defined code was using the built-in one.